src/program.js

```diff
@@ -521,6 +521,12 @@ Example: $0 --help run.
         },
       },
     )
+    .command(
+      'dump-config',
+      'Run config discovery and dump the resulting config data',
+      commands.dumpConfig,
+      {},
+    )
```

Comment:
I filed an issue to document this command eventually: https://github.com/mozilla/extension-workshop/issues/1784

Reply:
Thank you for taking care of filing that followup as in issue in the extension-workshop! :heart: